res/lang/stats/stats_pl.txt

```diff
@@ -0,0 +1,96 @@
+CREDIT_REPAID   = kredyt spłacony
+CREDIT_TAKEN    = wzięty kredyt
+ACCOUNT_BALANCE = Saldo konta
+TARGET_GROUPS   = Grupy docelowe
+
+ACHIEVEMENTS                        = Osiągnięcia
+ACHIEVEMENTCATEGORY_CATEGORIES      = Kategorie
+ACHIEVEMENTCATEGORY_ALL             = Wszystkie kategorie
+ACHIEVEMENTCATEGORY_NEWS            = Wiadomości
+ACHIEVEMENTCATEGORY_PROGRAMMES      = Programy
+ACHIEVEMENTCATEGORY_STATIONMAP      = Nadajniki
+ACHIEVEMENTCATEGORY_MISC            = Różne
+YOU_JUST_COMPLETED_ACHIEVEMENTTITLE = Właśnie ukończyłeś "%ACHIEVEMENTTITLE%".
+ACHIEVEMENT_COMPLETED               = Osiągnięcie ukończone
+REWARD                              = Nagroda
+TASK                                = Zadanie
+
+BEFORE_XTH_PLAYER_RESTART = Przed %X%. restartem gracza
+
+PLAYER_WENT_BANKRUPT                   = Gracz zbankrutował.
+PLAYER_X_FROM_CHANNEL_Y_WENT_BANKRUPT1 = Gracz %X% kanału %Y% nie grał wystarczająco dobrze i zbankrutował.
+PLAYER_X_FROM_CHANNEL_Y_WENT_BANKRUPT2 = Gracz %X% kanału %Y% miał nadzieję, że ma plan awaryjny, ale właśnie zbankrutował.
+PLAYER_X_FROM_CHANNEL_Y_WENT_BANKRUPT3 = Kanał %Y% zbankrutował. Stary menedżer %X% został zwolniony za niekompetencję.
+PLAYER_WILL_START_AGAIN                = Gracz rozpocznie grę od nowa.
+AI_WILL_TAKE_OVER_THIS_PLAYER          = SI przejmie kontrolę nad tym graczem.
+
+
+STATISTICS_NOT_AVAILABLE    = Statystyki nie są dostępne
+BROADCASTING_OUTAGE         = Przerwa w nadawaniu !!!
+ADVERTISMENT                = Reklama
+BLOCK                       = blok
+BLOCKS                      = bloki
+NEWS                        = Wiadomości
+SPECIAL_NEWS_BROADCAST      = Specjalne wiadomości
+PROGRAMME                   = Program
+TV_OFF                      = TV jest wyłączony!
+TV_TURN_IT_ON               = Powinieneś włączyć go ponownie ;D
+AUDIENCE_RATING             = Ocena widzów
+AUDIENCE_RATINGS            = Oceny widzów
+POTENTIAL_AUDIENCE_NUMBER   = Potencjalni widzowie
```

Comment:
This looks like, you did not start off with the current master version. The keys for the potential audience have slightly changed.

Reply:
please change according to the structure of the other language files and the fix proposed in PR https://github.com/TVTower/TVTower/pull/1139/files#diff-ed36ad9ad83c9396a726a4e4fdf8f6887f986745a1a814e2bd9e49f575d1bf99